modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/utils/rulesSupport.js

```diff
@@ -373,11 +373,22 @@ export const isActionsValid = (actions) =>
 				case 'auto-fill': {
 					const {inputs, outputs} = payload;
 
-					return (
-						Boolean(target) &&
-						(Boolean(Object.values(inputs).length) ||
-							Boolean(Object.values(outputs).length))
+					let valid =
+						Boolean(Object.values(inputs).length) ||
+						Boolean(Object.values(outputs).length);
```

Comment:
It can be simplified by converting  to `Boolean` only once:
```suggestion
						Boolean(Object.values(inputs).length) || Object.values(outputs).length);
```